Tasks/GithubReleaseV0/task.json

```diff
@@ -61,19 +58,16 @@
             "defaultValue": "$(Build.SourceVersion)",
             "required": true,
             "visibleRule": "action = Create || action = Edit",
```

Comment:
visibilityRules in this file also will need to be updated as the action keys have changed to lowercase